File: tys-application/tys-commons/src/main/java/co/com/mintrabajo/tys/commons/domain/DatosCondicionVinculacion.java
package co.com.mintrabajo.tys.commons.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
public class DatosCondicionVinculacion {
	
	private CondicionVinculacion condicionVinculacion;
	private Direccion direccionVinculo;
	private HorarioLaboral horarioLaboral;
	
	public DatosCondicionVinculacion(){
		
	}
}

File: tys-spa/ciudadano/src/app/domain/datos-condicion-vinculacion.ts
import {CondicionVinculacion} from './condicion-vinculacion';
import {HorarioLaboral} from './horario-laboral';
import {Direccion} from './direccion';

export interface DatosCondicionVinculacion {
  condicionVinculacion: CondicionVinculacion;
  direccionVinculo: Direccion;
  horarioLaboral: HorarioLaboral;
}
